src/catalog/database_catalog.cpp

```diff
@@ -329,6 +329,10 @@ void DatabaseCatalog::BootstrapPRIs() {
   const std::vector<col_oid_t> get_indexes_oids{postgres::INDOID_COL_OID};
   get_indexes_pri_ = indexes_->InitializerForProjectedRow(get_class_oid_kind_oids);
 
+  const std::vector<col_oid_t> get_live_indexes_oids{postgres::INDOID_COL_OID, postgres::INDISLIVE_COL_OID};
```

Comment:
See previous comment about liveness. I'm not sure we want this in a transactional catalog.